pages/community/events/index.js

```diff
@@ -0,0 +1,206 @@
+
+          if (i1.featured && !i2.featured) return -1;
+          if (!i1.featured && i2.featured) return 1;
+          return i2Date - i1Date;
+        })
+      : meetings
+  );
+
+  return (
+    <GenericLayout
+      title="AsyncAPI events"
+      description="Our catalogs of events and meetups"
+      image={image}
+      hideBanner={true}
+      wide
+    >
+      <div className="mt-10 sm:mt-28">
+        <div className="w-full sm:w-9/12">
+          <h1 className="countdown-text-gradient font-semibold text-3xl lg:text-7xl leading-tight md:text-4xl">
+            Join an AsyncAPI event from anywhere in the world.
+          </h1>
+          <div className="mt-10">
+            <GoogleCalendarButton href="https://calendar.google.com/calendar/u/3?cid=Y19xOXRzZWlnbG9tZHNqNm5qdWh2YnB0czExY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t" />
+            <ICSFileButton
+              href="https://calendar.google.com/calendar/ical/c_q9tseiglomdsj6njuhvbpts11c%40group.calendar.google.com/public/basic.ics"
+              className="mt-2 md:mt-0 md:ml-2"
+            />
+          </div>
+        </div>
+        <div className="flex justify-end mt-10">
+          <div className="w-full sm:w-1/3">
+            <Heading
+              level="h2"
+              typeStyle="body-lg"
+              textColor="text-gray-700"
+              className="text-slate-500"
+            >
+              All events/meetings are live streamed to all AsyncAPI social media
+              accounts. To learn more about meetings setup and automation
+              <TextLink
+                href="https://github.com/asyncapi/community/blob/master/MEETINGS_ORGANIZATION.md"
+                target="_blank"
+              >
+                read our FAQ
+              </TextLink>
+              .
+            </Heading>
+          </div>
+        </div>
+        <hr className="mt-20 border-dotted border-t-2 border-black " />
+      </div>
+      <div className="mt-24 w-full h-auto sm:h-[400px]">
+        <div className="w-full h-full bg-confBg rounded-md bg-cover bg-center p-10 flex flex-col justify-between text-white">
+          <img
+            src="/img/logos/confLogo.webp"
+            alt="conf-logo"
+            className="w-[100px] sm:w-[150px]"
+          />
+          <div className="w-full sm:w-[70%]">
+            <Heading
+              level="h2"
+              typeStyle="heading-sm-semibold"
+              className="mt-10 lg:text-2xl"
+            >
+              Watch the AsyncAPI 2022 conference recordings from anywhere around
+              the world for free
+            </Heading>
+            <a
+              href="https://www.youtube.com/playlist?list=PLbi1gRlP7pijRiA32SU36hD_FW-2qyPhl"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <div className="mt-5 flex items-center">
+                <span>Watch now</span>
+                <ArrowRightIcon className="w-[20px] ml-3 mt-1" />
+              </div>
```

Comment:
```suggestion
              <div className="mt-5 flex gap-3 items-center">
                <span>Watch now</span>
                <ArrowRightIcon className="w-[20px]" />
              </div>
```

Just a small suggestion because that arrow seemed to me quite misaligned as compared to the text. It is more top-aligned right now, than centred to the flex.